lib/Serialization/DeserializeSIL.cpp

```diff
@@ -430,8 +430,9 @@ SILDeserializer::readSILFunctionChecked(DeclID FID, SILFunction *existingFn,
     break;
     
   case SILStage::Lowered:
-    llvm_unreachable("cannot deserialize into a module that has entered "
-                     "Lowered stage");
+    if (!declarationOnly)
```

Comment:
I'm not sure if this is correct, because the large loadable types pass changes function signatures as well.

Reply:
It does change function signatures yes.